simple_mediation.rmd

```diff
@@ -0,0 +1,123 @@
+```{r, include=FALSE}
+options(tinytex.verbose = TRUE)
+```
+
+```{r}
+#RQ: Does domain-specifical movement behaviour profile membership mediate the relationship between socioeconomic position and socio-emotional outcomes in children?
+##Simple mediation for males
+```
+
+```{r include=FALSE}
+library(medflex)
+library(dplyr)
+library(haven)
+library(rmarkdown)
+```
+
+```{r}
+# Load male dataset from Study 2, LSAC wave 6 data, and LSAC wave 8 data
+load(file = "Z:/LSAC dataset/Study_2/Study_2/df_males_domsp_weekday.RData")
+lsac_wave6 <- read_sas("Z:/LSAC dataset/General Release/Survey data/SAS/lsacgrb10.sas7bdat")
+lsac_wave8 <- read_sas("Z:/LSAC dataset/General Release/Survey data/SAS/lsacgrb14.sas7bdat")
+```
+
+```{r}
+# Select relevant columns from male dataset (hicid, profile membership, age, remoteness, Indigenous status, SEP)
```

Comment:
:bulb: From the tidyverse style guide:
> In code, use comments to explain the “why” not the “what” or “how”. 

and

> In data analysis code, use comments to record important findings and analysis decisions. If you need comments to explain what your code is doing, consider rewriting your code to be clearer.

That is, your comments don't need to repeat obvious things in code. It's obvious from the code what is being selected.

(I know this is silly, but the rules around writing concisely apply within code too)